src/openapi.json

```diff
@@ -103,7 +103,7 @@
           },
           "version": {
             "type": "integer",
-            "example": 1
```

Comment:
There is still value in providing an example (since the schema is used for GET as well and the example populates sample responses)

Reply:
Got it. I put it back in.